pages/installing-packages.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: page
+title: Installing Python packages with conda-forge
+permalink: /installing-packages
+nav_order: 35
+---
+
+# Avoiding the spaghetti installation
+
+Many ways to install Python and Python packages have been developed over the years, and not all of them are compatible.
+
+If you're having trouble installing Scikit-HEP packages (e.g. `pip install` fails with an error or you get an `ImportError`/`ModuleNotFoundError`), try following the instructions below to set up a conda-forge environment.
```

Comment:
```suggestion
If you're having trouble installing Scikit-HEP packages (e.g. `pip install` fails with an error or you get an `ImportError`/`ModuleNotFoundError`), try following the instructions below to set up a conda-forge environment using `mamba`
```

mention it up front in case people close the page thinking they already have conda / pyenv setup.